project.sh

```diff
@@ -451,9 +457,14 @@ npm-cmd () {
 	abort-on-error 'snapshotting'
 	$reset_cmd
 	abort-on-error 'resetting version'
-	if ! is-ci && cmd-exists clean;then
-		clean
+	local-clean
+}
+
+-install () {
+	if is-lein;then
+		lein-install install
 	fi
+	local-clean
```

Comment:
Clean after install, otherwise it tries to use compiled classes for subsequent runs.
e.g. `lein install;lein test` can fail